apps/kafka-mapper-microfrontend/src/components/App/App.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect } from "react";
+import { Typography, Grid, Paper, Divider } from "@mui/material";
+import { DndProvider } from "react-dnd";
+import { HTML5Backend } from "react-dnd-html5-backend";
+import { FhirResourcesLoader } from "../FhirResourcesLoader/FhirResourcesLoader";
+import { FhirMapperConfigEditor } from "../ConfigEditor/ConfigEditor";
+import {
+  FhirMapperConfigProvider,
+  useFhirMapperConfig,
+} from "../FhirMapperConfigProvider";
+
+export function Workspace() {
+  const { activeFhirResource } = useFhirMapperConfig();
+  return (
+    <Grid container spacing={3}>
+      <Grid item xs={12} md={6}>
+        <FhirResourcesLoader />
+      </Grid>
+      <Grid item xs={12} md={6}>
+        {activeFhirResource && <FhirMapperConfigEditor />}
+      </Grid>
+    </Grid>
+  );
+}
+
+export default function App(props) {
+  useEffect(() => {
+    const handleBeforeUnload = (event) => {
+      event.preventDefault();
+      // Custom logic to handle the refresh
+      // Display a confirmation message or perform necessary actions
```

Comment:
Pending?

Reply:
Yes, this should considred in the future to improve the error handling